src/Components/App/App.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+// import { Clock } from '../Clock/Clock';
+import { Clock } from '../Clock/Clock';
```

Comment:
```suggestion
import { Clock } from '../Clock';
```
імпортуємо з папки якщо вже є індекс